lib/policies/rate-limit/rate-limit.js

```diff
@@ -1,13 +1,12 @@
 let RateLimit = require('express-rate-limit');
-const logger = require('../../logger').logPolicy;
```

Comment:
In order to work It should be 
const logger = require('../../logger').policy

https://github.com/ExpressGateway/express-gateway/blob/63ee70aa829c22ab2651b3e6fbe6b64f70647220/lib/policies/key-auth/keyauth.js#L4